src/main/java/frc/robot/subsystems/gripper/Gripper.java

```diff
@@ -0,0 +1,73 @@
+package frc.robot.subsystems.gripper;
+
+import edu.wpi.first.networktables.BooleanSubscriber;
+import edu.wpi.first.networktables.DoubleSubscriber;
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Ports;
+import frc.robot.subsystems.LoggedSubsystem;
+
+public class Gripper extends LoggedSubsystem<GripperLoggedInputs> {
+    private GripperLoggedInputs gripperLoggedInputs;
```

Comment:
There is no need for this, you have thus from the super class

Reply:
done.